test/bin/mdx-test/expect/block-type/test-case.md.expected

```diff
@@ -0,0 +1,23 @@
+It is possible to explicitly state the type of a block using the
+`block-type` label, working around language header and content based
+inference which can sometime lead to troublesome error messages.
+
+The following blocks use a volontarily misleading language header that would
+normally lead to errors if we let MDX infer the type of block based on them.
+
+```ocaml block-type=toplevel
+# 1 + 1;;
+- : int = 2
+```
+
+```ocaml block-type=ocaml
```

Comment:
This seems a bit verbose? Is `ocaml` the shortcut for `ocaml block-type=ocaml` ?